web/src/routes/(user)/map/+page.svelte

```diff
@@ -35,6 +36,9 @@
   });
 
   $: $featureFlags.map || goto(AppRoute.PHOTOS);
+  const omit = (obj: MapSettings, key: string) => {
```

Comment:
I don't know if it's a vite or a lodash issue, I haven't investigated further than that.
```ts
"omit" is imported from external module "lodash-es" but never used in "src/lib/components/elements/dropdown.svelte", "src/lib/components/admin-page/settings/admin-settings.svelte", "src/lib/stores/assets.store.ts", "src/lib/utils/timeline-util.ts", "src/lib/components/admin-page/settings/ffmpeg/ffmpeg-settings.svelte", "src/lib/components/admin-page/settings/machine-learning-settings/machine-learning-settings.svelte", "src/lib/components/admin-page/settings/map-settings/map-settings.svelte", "src/lib/components/admin-page/settings/thumbnail/thumbnail-settings.svelte", "src/lib/components/admin-page/settings/server/server-settings.svelte", "src/lib/components/admin-page/settings/trash-settings/trash-settings.svelte", "src/lib/components/admin-page/settings/new-version-check-settings/new-version-check-settings.svelte", "src/lib/components/admin-page/settings/library-settings/library-settings.svelte", "src/lib/components/admin-page/settings/logging-settings/logging-settings.svelte", "src/lib/components/admin-page/settings/job-settings/job-settings.svelte", "src/lib/components/admin-page/settings/password-login/password-login-settings.svelte", "src/lib/components/admin-page/settings/oauth/oauth-settings.svelte", "src/lib/components/admin-page/settings/storage-template/storage-template-settings.svelte", "src/lib/components/admin-page/settings/theme/theme-settings.svelte", "src/routes/(user)/albums/+page.svelte" and "src/routes/(user)/map/+page.svelte".
```
Generally speaking, I prefer to avoid using lodash

Reply:
Sounds good to me. I'll file an issue with Vite, but we might as well go with this in the meantime.

Reply:
I just filed an issue over in the Vite repo for this: https://github.com/vitejs/vite/issues/15890